webapp/src/components/expanded_view/reaction_button.tsx

```diff
@@ -53,9 +53,10 @@ export const ReactionButton = forwardRef(({trackEvent}: Props, ref) => {
 
     const handleUserPicksEmoji = (ecd: EmojiClickData) => {
         const emojiData: EmojiData = {
-            name: ecd.emoji,
+            name: ecd.names[0].replaceAll(' ', '_'),
```

Comment:
Just confirming that `ecd.names` is guaranteed to be non-empty, right? It seems like it by judging on the code below, but double checking.

Reply:
Hah, good eye. I did look at the docs and there was nothing stated either way, so I looked at the `emojis.json` and I searched for an empty names list and didn't find one. So I assumed no. But now that you also mention it, it made me worried. So just in case I'll do a test first. :)